File: C++/measurement_converter.cpp
#include <iostream>

using namespace std;

int main()
{
	double feet, yards, miles, ounces, pounds;

	int choice = 0;
	while (choice != -1)
	{
		cout << "Do you want to 1=Convert feet to yards, 2=Convert feet to miles, or 3=Convert ounces to pounds? \n";
		cout << "-1 to quit\n";
		cin >> choice;

		if (choice == 1)
		{
			cout << "How many feet? > ";
			cin >> feet;

			yards = feet / 3.0;
			cout << "Yards: " << yards << endl;
		}
		else if (choice == 2)
		{
			cout << "How many feet? > ";
			cin >> feet;

			miles = feet / 5280;
			cout << "Miles: " << miles << endl;
		}
		else if (choice == 3)
		{
			cout << "How many ounces? > ";
			cin >> ounces;

			pounds = ounces / 16;
			cout << "Pounds: " << pounds << endl;
		}
		else if (choice == -1)
		{
			cout << "Ok, bye\n";
		}
		else
		{
			cout << "Invalid choice\n";
		}
	}
	return 0;
}

File: Csharp/measurement_converter.cs


namespace MyApp
{
    internal class measurement_converter
    {
        static void Main(string[] args)
        {
            //feet to meters
            string i;
            Console.WriteLine("A to convert feet to meters");
            Console.Write("B to convert miles to kilometers > ");
            i = Console.ReadLine();
            if (i == "a" || i == "A") 
            {
                string s;
                Console.Write("Convert how many feet? > ");
                s = Console.ReadLine();
                double feet = Double.Parse(s);
                double meters = feet * 12 * 2.54 / 100;
                Console.WriteLine($"{feet} Feet = {meters} Meters");
            }else if(i == "b" || i == "B") 
            {
                string x;
                Console.Write("Convert how many miles? > ");
                x = Console.ReadLine();
                double miles = Double.Parse(x);
                double kilometers = miles * 1.609344;
                Console.WriteLine($"{miles} Miles = {kilometers,0:0.#} Kilometers");
            }
        }
    }
}
